src/main/java/org/jabref/gui/maintable/RightClickMenu.java

```diff
@@ -106,11 +106,8 @@ private static Menu createCopySubMenu(ActionFactory factory,
         if (previewPreferences.getSelectedPreviewLayout() instanceof CitationStylePreviewLayout) {
             copySpecialMenu.getItems().add(factory.createMenuItem(StandardActions.COPY_CITATION_HTML, new CopyCitationAction(CitationStyleOutputFormat.HTML, dialogService, stateManager, clipBoardManager, taskExecutor, previewPreferences)));
             Menu copyCitationMenu = factory.createMenu(StandardActions.COPY_CITATION_MORE);
-            copyCitationMenu.getItems().addAll(
-                    factory.createMenuItem(StandardActions.COPY_CITATION_TEXT, new CopyCitationAction(CitationStyleOutputFormat.TEXT, dialogService, stateManager, clipBoardManager, taskExecutor, previewPreferences)),
-                    factory.createMenuItem(StandardActions.COPY_CITATION_RTF, new CopyCitationAction(CitationStyleOutputFormat.RTF, dialogService, stateManager, clipBoardManager, taskExecutor, previewPreferences)),
-                    factory.createMenuItem(StandardActions.COPY_CITATION_ASCII_DOC, new CopyCitationAction(CitationStyleOutputFormat.ASCII_DOC, dialogService, stateManager, clipBoardManager, taskExecutor, previewPreferences)),
-                    factory.createMenuItem(StandardActions.COPY_CITATION_XSLFO, new CopyCitationAction(CitationStyleOutputFormat.XSL_FO, dialogService, stateManager, clipBoardManager, taskExecutor, previewPreferences)));
+            copyCitationMenu.getItems().add(
```

Comment:
Can we add `COPY_CITATION_HTML`? - This is IMHO a proper replacement for RTF, ASCII_DOC, XSLFO

    factory.createMenuItem(StandardActions.COPY_CITATION_HTML, new CopyCitationAction(CitationStyleOutputFormat.HTML, dialogService, stateManager, clipBoardManager, taskExecutor, previewPreferences)),

Reply:
See two lines above, HTML has been already available

Reply:
OK - I did not expect that being present in a different menu.

Can we merge it into one? The current sub menu has only one item!

![grafik](https://user-images.githubusercontent.com/1366654/151559744-c2bed5d8-7d19-4738-a021-98fa94c51c0c.png)

New:

![grafik](https://user-images.githubusercontent.com/1366654/151562328-863443e3-d32b-4ff1-87ff-b1588435692c.png)